File: Projects/CppProjects/calculator/opp.cpp
#include "main.h"

int add(int a, int b)
{
    return (a + b);
}

int sub(int a, int b)
{
    return (a - b);
}

int mul(int a, int b)
{
    return (a * b);
}

int divd(int a, int b)
{
    if (b == 0)
    {
        Excp_0 zero;
        zero.a = a;
        zero.b = b;
        throw zero;
    }
    return (a / b);
}

int mod(int a, int b)
{
    if (b == 0)
    {
        Excp_0 zero;
        zero.a = a;
        zero.b = b;
        throw zero;
    }
    return (a % b);
}

File: PythonProgramming/Module/opp.py
def add(x: int, y: int):
    return x + y

def sub(x: int, y: int):
    return x - y

def mul(x: int, y: int):
    return x * y

def div(x: int, y: int):
    return x / y

def mod(x: int, y: int):
    return x % y

if __name__ == "__main__":
    from sys import argv
    print(str(add(int(argv[1]), int(argv[2]))))
